test/bdd/features/verifier_profile_rest_api.feature

```diff
@@ -0,0 +1,244 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@verifier_profile_rest
+Feature: Verifier Profile Management REST API
+  Scenario: Create a new verifier profile
+    When HTTP POST is sent to "http://localhost:8075/verifier/profiles"
+         """
+         {
+           "name": "verifier profile",
+           "url": "https://verifier.example.com",
+           "checks": {
+             "credential": {
+               "proof": true,
+               "format": [
+                 "jwt_vc"
+               ],
+               "status": true
+             },
+             "presentation": {
+               "proof": true,
+               "format": [
+                 "jwt_vp"
+               ]
+             }
+           },
+           "organizationID": "org1"
+         }
+         """
```

Comment:
I do not suggest these kind of tests for VCS bdd tests.
Feature file should contain usecase based stories which should be easy to understand by non-technical audiences too.

Reply:
for API coverage we should have postman/openAPI based automation tests.